tests/monotouch-test/MetalPerformanceShaders/MPSImageHistogramSpecificationTest.cs

```diff
@@ -26,14 +24,18 @@ public class MPSImageHistogramSpecificationTest
 		[Test]
 		public void Constructors ()
 		{
+#if !MONOMAC
 			TestRuntime.AssertDevice ();
 			TestRuntime.AssertXcodeVersion (7, 0);
+#else
+			TestRuntime.AssertXcodeVersion (9, 0);
+#endif
 
 			MPSImageHistogramInfo info = new MPSImageHistogramInfo ();
 			info.NumberOfHistogramEntries = 256;
 			using (var obj = new MPSImageHistogramSpecification (MTLDevice.SystemDefault, ref info)) {
 				var rv = obj.HistogramInfo;
-				Asserts.AreEqual (info, rv, "HistogramForAlpha");
+				Assert.That(info, Is.EqualTo(rv), "HistogramForAlpha");
```

Comment:
What's the difference between the new and the old assert? It seems to me these are identical (and thus the change is not needed).

There's also the matter of missing spaces :smile:

Reply:
Asserts.AreEqual gives me an error on mac:

/Users/Tim/Projects/Xamarin/xamarin-macios-xcode9/xamarin-macios/tests/monotouch-test/MetalPerformanceShaders/MPSImageHistogramSpecificationTest.cs(39,13): error CS1501: No overload for method 'AreEqual' takes 2 arguments
/Users/Tim/Projects/Xamarin/xamarin-macios-xcode9/xamarin-macios/tests/monotouch-test/MetalPerformanceShaders/MPSImageHistogramEqualizationTest.cs(38,23): error CS1503: Argument 1: cannot convert from 'MetalPerformanceShaders.MPSImageHistogramInfo' to 'bool'
/Users/Tim/Projects/Xamarin/xamarin-macios-xcode9/xamarin-macios/tests/monotouch-test/MetalPerformanceShaders/MPSImageHistogramEqualizationTest.cs(38,29): error CS1503: Argument 2: cannot convert from 'MetalPerformanceShaders.MPSImageHistogramInfo' to 'bool'

Reply:
Then you need to fix the `#if` here:

https://github.com/xamarin/xamarin-macios/blob/8d1ebd48e5bc967534b1be5b07f98e388fa25859/tests/monotouch-test/Asserts.cs#L183-L191